pkg/serverless/trace/propagation/carriers.go

```diff
@@ -201,6 +213,51 @@ func snsEntityCarrier(event events.SNSEntity) (tracer.TextMapReader, error) {
 	return carrier, nil
 }
 
+// handleEventBridgeThroughSNS builds payload carrier for an EventBridge
+// event wrapped in an SNS message.
+func handleEventBridgeThroughSNS(eventBridgeEvent map[string]interface{}) (tracer.TextMapReader, error) {
+	detail, ok := eventBridgeEvent["detail"].(map[string]interface{})
+	if !ok {
+		return nil, errorUnsupportedPayloadType
+	}
+
+	datadogInfo, ok := detail["_datadog"].(map[string]interface{})
```

Comment:
use the constant instead of `_datadog` string

Reply:
Good catch